### PR TITLE
feat: add more favicons

### PR DESCRIPTION
**Pull request by spalt08**
- Minor `server.ts` refactoring
- Added default fallback website favicon
- Added `favicon.ico`
- Changed sharing image

### CODE REVIEW

File: packages/web/src/components/Layout.tsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ export function Layout({
         <meta charSet='utf-8' />
         <meta name='viewport' content='width=device-width, initial-scale=1' />
         <meta name='mobile-web-app-capable' content='yes' />
+        <link rel='icon' type='image/x-icon' href='/static/favicon.ico' />
         {/* TODO: preload all fonts and add icons after redesign
           <link rel="preload" href={} as="font" type="font/woff2"/>
           <link rel='icon' type='image/svg+xml' href={'TODO'} />
```
Comment (ctizen) [inline packages/web/src/components/Layout.tsx:46]: Looks like it's time to do something with this TODO

Comment (spalt08) [reply packages/web/src/components/Layout.tsx:46]: I've created a ticket for that

**Review by ctizen — APPROVED** [review]

File: packages/web/src/server.tsx [hunk 2]
```diff
@@ -13,41 +13,110 @@ import {
...
+        }
+
+        if (asset.endsWith('.css')) {
+          acc.css.push(path.join(publicStaticDir, asset));
+        }
+
+        return acc;
+      }, assets);
+  } catch (e) {
+    return assets;
```
Comment (zardak) [inline packages/web/src/server.tsx:110]: nit: Seems like we'd rather throw here than return empty result?

File: packages/web/src/server.tsx [hunk 3]
```diff
@@ -13,41 +13,110 @@ import {
...
+  }
+}
+
+async function pipeS3Object(objectKey: string, res: Response, fallback?: Buffer) {
+  try {
+    if (isDevelopment()) {
+      throw new Error('Development mode');
```
Comment (zardak) [inline packages/web/src/server.tsx:117]: I'm rather unsure about this. Why not just let errors happen naturally?

File: packages/web/src/server.tsx [hunk 4]
```diff
@@ -66,7 +135,9 @@ async function pipeS3Object(objectKey: string, res: Response) {
       throw new Error('Unexpected response format');
     }
   } catch (e) {
-    if (e instanceof NoSuchKey) {
+    if (fallback) {
+      res.send(fallback);
```
Comment (zardak) [inline packages/web/src/server.tsx:139]: As we discussed, placing the fallback client-side would save us and users minor amount of traffic. And, more so, be a step towards correct implementation.

**Review by zardak — APPROVED** [review]

**Pull request merged**